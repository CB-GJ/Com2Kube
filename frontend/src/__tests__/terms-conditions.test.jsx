import React from "react"
import renderer from "react-test-renderer"
import TermsConditions from "../pages/not-found"

describe("Terms and Conditions", () => {
  it("renders correctly", () => {
    const wrapper = renderer.create(<TermsConditions />).toJSON()
    expect(wrapper).toMatchSnapshot()
  })
})
