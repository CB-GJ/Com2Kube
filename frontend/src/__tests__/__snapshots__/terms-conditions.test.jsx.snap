// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terms and Conditions renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <h2
    className="MuiTypography-root MuiTypography-h2"
  >
    Oops..
  </h2>
  <h4
    className="MuiTypography-root MuiTypography-h4"
  >
    We can't seem to find the page you're looking for.
  </h4>
  <img
    alt="not found error img"
    className="makeStyles-img-2"
    src="test-file-stub"
  />
</div>
`;
